src/core/hash.js

```diff
@@ -14,14 +14,13 @@ let cache = {};
  * @returns {String}
  */
 let stringify = (data) => {
-    let out = '';
-
-    for (let p in data) {
-        let val = data[p];
-        out += p + (typeof val == 'object' ? stringify(data[p]) : data[p]);
+    if (typeof data == 'object') {
+        let out = '';
+        for (let p in data) out += p + stringify(data[p]);
+        return out;
+    } else {
```

Comment:
no need for else here? (return statement in if branch)

Reply:
I did compare this. That's just a coding style thing, and I was focusing on what ultimately is output (which is likely nested ternaries on a single statement)

Happy to change this depending on what this repo prefers given that Terser will likely output the same code for either 👍

Reply:
thank you :)

Reply:
Honestly if prettier didn't complain or change it 😄  then it's totally fine by me! Good find by the way!

Reply:
https://eslint.org/docs/rules/no-else-return

;)